src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1136,7 +1136,24 @@ class ApplicationManagerImpl
    */
   protocol_handler::MajorProtocolVersion SupportedSDLVersion() const OVERRIDE;
 
+  void ApplyFunctorForEachPlugin(
+      std::function<void(plugin_manager::RPCPlugin&)> functor) OVERRIDE;
+
  private:
+  /**
+   * @brief Adds application to registered applications list and marks it as
+   * registered
+   * @param application Application that should be added to registered
+   * applications list.
+   */
+  void AddAppToRegisteredAppList(ApplicationSharedPtr application);
+
+  /**
+   * @brief Adds all extensions to applications
+   * @param application Application to which will be added extensions
+   */
+  void AddExtensionsToApp(ApplicationSharedPtr application);
```

Comment:
can be const?

Reply:
@alexkutsan Changed in 9631f38080e906c4b63be2d3e2c449628e284ccf